fuzz_lightyear/supplements/setup.py

```diff
@@ -0,0 +1,23 @@
+from typing import Callable
+
+from fuzz_lightyear.datastore import get_setup_fixtures
+
+
+def setup(func: Callable) -> Callable:
+    """
+    Use the @setup decorator to mark functions that you'd like execute prior
+    to running fuzz-lightyear tests
+
+    Decorated functions should be in the file along with your factory fixtures
+
+    Basic use:
+
+        >>> import fuzz lightyear
+        >>> @fuzz_lightyear.setup
+        ... def setup_config():
+        ...     // do configuration here
+
+    """
+    setup_fixtures = get_setup_fixtures()
+    setup_fixtures.append(func)
```

Comment:
Why do we need a list here? Can we just only support one function?

Reply:
I thought the data structure model & a "getter" function was more consistent with the other fixtures as opposed to having a sole global variable where the function was stored.

Reply:
Gotcha. That makes sense.

Using a list for multiple setup subscriptions is weird, but if it's a way to avoid globals, that makes sense. Probably worth recording that rationale down, otherwise, 🚢